numba/core/runtime/context.py

```diff
@@ -372,7 +376,27 @@ def _call_incref_decref(self, builder, typ, value, funcname):
             # function be called with a NULL pointer?
             fn.args[0].add_attribute("noalias")
             fn.args[0].add_attribute("nocapture")
-            builder.call(fn, [mi])
+
+            trace_str = ""
+
+            if config.DEBUG_NRT and config.DEBUG_NRT_STACK_LIMIT:
+                trace = io.StringIO()
+                traceback.print_stack(limit=config.DEBUG_NRT_STACK_LIMIT+2, file=trace)
+                # The last two stack frames are `_call_incref_decref`
+                # and `traceback.print_stack` which we ignore.
+                clean_trace = trace.getvalue().split('\n')[:-5]
```

Comment:
If the trace format changes `-5` might not be appropriate, should this instead scan backwards for the `_call_incref_decref` name?

Reply:
Addressed in 170f84f